static/app/components/nav/index.spec.tsx

```diff
@@ -0,0 +1,104 @@
+import {LocationFixture} from 'sentry-fixture/locationFixture';
+import {OrganizationFixture} from 'sentry-fixture/organization';
+import {UserFixture} from 'sentry-fixture/user';
+
+import {render, screen} from 'sentry-test/reactTestingLibrary';
+
+import Nav from 'sentry/components/nav';
+import ConfigStore from 'sentry/stores/configStore';
+import type {Organization} from 'sentry/types/organization';
+import {useLocation} from 'sentry/utils/useLocation';
+
+jest.mock('sentry/actionCreators/account');
+jest.mock('sentry/utils/useServiceIncidents');
+jest.mock('sentry/utils/useLocation');
```

Comment:
i'm not sure you need to mock useLocation if you aren't setting any params. Would prefer people mock location via the router context in our render function

https://github.com/getsentry/sentry/blob/master/tests/js/sentry-test/reactTestingLibrary.tsx#L63-L71

Reply:
here's a really small example

https://github.com/getsentry/sentry/blob/master/static/app/components/modals/commandPalette.spec.tsx#L91-L93

Reply:
Thanks for the test feedback! These have been greatly simplified in [7760e83](https://github.com/getsentry/sentry/pull/77665/commits/7760e834027dfd6ff9998f29be7056469ca5908e).